packages/compass-e2e-tests/tests/in-use-encryption.test.ts

```diff
@@ -126,12 +103,8 @@ describe('FLE2', function () {
       }
 
       await delay(10000);
-
-      console.log('Read favorites after disconnecting and delay');
-      await readSavedConnectionsFolder(compass);
```

Comment:
Is removing these things intentional?

Reply:
Yeah I used this previously to check if recents are actually exist in the folder, and haven't removed it after.